README.adoc

```diff
@@ -22,14 +22,26 @@ buildPlugin()
 
 ==== Optional arguments
 
-* `jdkVersions` (default: `[8]`) - JDK version numbers, must match a version
-  number jdk tool installed
 * `repo` (default: `null`  inherit from Multibranch) - custom Git repository to check out
 * `failFast` (default: `true`) - instruct Maven tests to fail fast
 * `platforms` (default: `['linux', 'windows']`) - Labels matching platforms to
   execute the steps against in parallel
+* `jdkVersions` (default: `[8]`) - JDK version numbers, must match a version
+    number jdk tool installed
 * `jenkinsVersions`: (default: `[null]`) - a matrix of Jenkins baseline versions to build/test against in parallel (null means default,
   only available for Maven projects)
+* `configurations`: An alternative to `platforms`, `'jdkVersions'` and `'jenkinsVersions'` arguments. Those 3 form a cartesian
```

Comment:
would be nice to explicitly say that the option is not compatible with platforms/jdkVersions/jenkinsVersions